src/App.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import './i18n';
-
+import { HashRouter, Route } from 'react-router-dom';
 function App() {
   return (
-    <div>
-      <h1>Online Books</h1>
-    </div>
+    <HashRouter>
```

Comment:
why did you used HashRouter ?

Reply:
Because it is easier to host on GitHub in case we needed that and it assure saving the state more than the normal route

Reply:
for the first reason there is some fixes we can follow 
anyway if you are sure about using this I will merge it

Reply:
Let's go with this it is safer I believe